ports/rp2/Makefile

```diff
@@ -34,26 +34,26 @@ ifeq ($(BUILD_VERBOSE),1)
 MAKE_ARGS += VERBOSE=1  # Picked up in Makefile generated by CMake
 endif
 
-CMAKE_ARGS += -DMICROPY_BOARD=$(BOARD) -DMICROPY_BOARD_DIR="$(abspath $(BOARD_DIR))"
+override CMAKE_ARGS += -DMICROPY_BOARD=$(BOARD) -DMICROPY_BOARD_DIR="$(abspath $(BOARD_DIR))"
```

Comment:
I really don't think it's a good idea to complicate the Makefile by adding `override`.  Combining make and cmake is already tricky enough.

Why exactly is this needed, what were you trying to achieve with it?  Can we do it a different way, eg just call cmake directly in such cases?